modules/ROOT/pages/entities-overview.adoc

```diff
@@ -0,0 +1,166 @@
+
+== Before You Begin
+
+Before getting started, ensure that you have:
+
+* An Anypoint Platform account that has the Monitoring Viewer permission and permissions required to view apps and APIs.
+* A supported version of Mule Runtime engine.
+
+== Supported Versions of Mule Runtime Engine
+
+include::partial$include-supported-mule-runtime-version.adoc[]
+
+== Access the Entities Overview Page
+
+include::partial$include-nav-steps-no-permissions.adoc[]
+
+include::partial$include-nav-entities.adoc[]
+
+The *Entities Overview* page appears, showing a summary of data for entities in the current business group and environment.
+
+== Search for Entities
+
+The search bar in the *Entities Overview* page enables you to create search parameters to curate different assortments of entity data on the page. You can add and remove attributes such as status, errors, business groups, and environments to further refine your results. Adding different search parameters changes data in the summary, which is refreshed each time you conduct a new search. 
+
+By default, the *Entity Overview* page shows data for entities in your current business group and environment. You can change or remove these parameters to see data for entities in different environments or business groups by clicking the *X* next to each option.
+
+The search bar also offers quick filters, such as `Top 10 entities with highest error volume`, to show you common scenarios at a glance. You can also combine these with attributes and metrics to further specify the content that appears. 
+
+You can curate your search results for entity data using the following attributes: 
+
+* Environment 
+* Entity name 
+* Entity type 
+* Status
+* Response time 
+* Requests
+* Errors 
+* Client App (for APIs)
+* Locations (for APIs)
+* Org ID 
+
+You can curate your search results for entity data using the following metrics:
+
+tbd
+
+=== Applying Search Parameters
+
+include::partial$include-nav-steps-no-permissions.adoc[]
+
+include::partial$include-nav-entities.adoc[]
+
+. In the search bar, enter an attribute, element, business group, or environment you want to use to filter your data.
+
+=== Example of Applied Search Parameters
+
+tbd
+
+== Filtering Data using the Time Period Selector
+
+You can display the data points collected for the last given period of time (such as the last 5 or 30 minutes) or over a given date and time range. 
+
+Organizations with a Gold, Platinum, or Starter subscription can view up to 30 days of data. Organizations with a Titanium or Advanced subscription can view up to one year of data.
+
+=== Setting the Time Period to Display
+
+Use the drop-down in the calendar icon to select the time period to display in a graph.
+
+Your setting applies across all charts in the entity summary. 
+
+=== Changing the Time Period to Display
+
+You can display an earlier or later time period in your chart, and you can zoom in or out to cover a shorter or longer period of time.
```

Comment:
right now we don't support zoom-in/zoom-out functionality on the date picker.

Reply:
```suggestion
You can display an earlier or later time period in your chart to cover a shorter or longer period of time.
```